CHANGELOG.md

```diff
@@ -1,13 +1,9 @@
-x.y.z (TBD)
+1.1.0 (2017-03-02)
 ------------------
 
-### Breaking Changes
-
 ### Enhancements
 - Added Azure Active Directory (AzureAD) credentials provider. (#1254)
 
```

Comment:
The weaver is something we haven't released, so the fixes are not visible. But I'll add a mention that we're using a new build system.